src/components/AnnotationPopover/AnnotationPopover.scss

```diff
@@ -76,33 +76,39 @@
         }
     }
 
-    .ba-annotation-label {
-        line-height: 23px;
-        padding: 3px 0;
-    }
-
-    .ba-annotation-label ~ .ba-action-controls {
-        border-left: 1px solid $see-see;
-        margin-left: 10px;
+    .ba-buffer {
+        display: none;
     }
 
     // MOBILE & TABLET CSS
     @media #{$mobile}, #{$tablet} {
-        .ba-popover-caret {
+        .ba-mobile-header {
+            align-self: flex-start;
```

Comment:
can this be achieved with just align-items or justify-content? They seem to be slightly more supported